tests/integration/updateTechRecordStatus.intTest.ts

```diff
@@ -114,7 +114,8 @@ describe("UpdateTechRecordStatus", () => {
                 })
                 .expectResolve((result: any) => {
                     expect(result.statusCode).toBe(404);
-                    expect(JSON.parse(result.body)).toBe(HTTPRESPONSE.RESOURCE_NOT_FOUND);
+                    // FIXME: array to string
```

Comment:
Not sure what ticket we created for it yet (assuming it is for VTM) but if not done yet we can copy and paste this reference in the ticket so we know what to search for but very good call for the tagging